package.json

```diff
@@ -22,11 +22,17 @@
     "isomorphic"
   ],
   "author": "Reza Akhavan <jedireza@gmail.com> (http://reza.akhavan.me/)",
+  "contributors": [
+    "Tane Piper <piper.tane@gmail.com> (https://github.com/tanepiper)"
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/jedireza/hapi-react-views/issues"
   },
   "homepage": "https://github.com/jedireza/hapi-react-views",
+  "engines": {
+    "node": ">=4.0.0"
+  },
```

Comment:
Again happy to defer on this. I think it's useful to endure that it's made explicit this module is node >4 only but if hapijs 11 is specified in peer dependencies then it's probably not so much of an issue. It adds additional useful data to npm when viewing the module there but not precious to keep it.